spec/dynamoid/persistence_spec.rb

```diff
@@ -846,10 +834,10 @@ def reload_address
         a1.save!
 
         a2.destroy(:skip_lock_check => true)
-        expect(Address.find(address.id)).to eql nil
+        expect { Address.find(address.id) }.to raise_exception(Dynamoid::Errors::RecordNotFound)
       end
 
-      it 'uses the correct lock_version even if it is modified' do
+      it 'uses the./spec/dynamoid/persistence_spec.rb correct lock_version even if it is modified' do
```

Comment:
Did you mean to add `./spec/dynamoid/persistence_spec.rb` here?

Reply:
Oops, good catch, dang